www/mixpanel.js

```diff
@@ -39,6 +39,8 @@ mixpanel.identify = function(id, onSuccess, onFail) {
 };
 
 mixpanel.init = function(token, onSuccess, onFail) {
+  this['__loaded'] = true;
```

Comment:
I'll merge and take care of this on master. ty for the raising the issue and contributing the pr 👍